packages/data-context/src/actions/ProjectActions.ts

```diff
@@ -89,6 +89,13 @@ type SetForceReconfigureProjectByTestingType = {
 const debug = debugLib('cypress:data-context:ProjectActions')
 
 export class ProjectActions {
+  /**
+   * @var globalLaunchCount
+   * Used as a read-only in the launchpad to ensure
+   * that launchProject is only called once if
+   * the --browser flag is passed alone.
+   */
+  private globalLaunchCount = 0
```

Comment:
This is only used to increment in `launchProject` and returned with a getter to use in gql-LocalSettings.ts